packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -67,6 +67,26 @@ const normalizeACF = entities =>
 
 exports.normalizeACF = normalizeACF
 
+// Combine all ACF Option page data
+exports.combineACF = function(entities) {
+ let acfOptionData = {}
+ // Map each ACF Options object keys/data to single object
+ _.forEach(entities.filter((e) => e.__type === `wordpress__acf_options`), (e) => {
+   if(e[`acf`]) {
+     Object.keys(e[`acf`]).map((k) => acfOptionData[k] = e[`acf`][k])
```

Comment:
I'm not sure if I'm reading this correctly - but if we have 2 option pages both with same fieldnames, one would get overwritten? Perhaps we should have structure like this:
```
optionsNode
├─ optionsPageIdName1
|  └─ title (example field)
└─ optionsPageIdName2
   └─ title
```
(ignore this if I read this code wrong)

Reply:
@pieh Thank you for the feedback.

I've pushed up a new commit that addresses both items in your review.

First, the ACF to REST version is now detected from the routes found. 

Secondly, I changed the structure to fit what you suggested. I neglected to think about duplicate field names. 

Now you would query like such:

```  query optionsQuery {
    allWordpressAcfOptions {
      edges {
        node{
          option_page_1 {
            test_acf
          }
          options {
            test_acf
          }
        }
      }
    }
  }
```

which would result in:
```
{
  "data": {
    "allWordpressAcfOptions": {
      "edges": [
        {
          "node": {
            "option_page_1": {
              "test_acf": "Test Text 2"
            },
            "options": {
              "test_acf": "Test Text 1"
            }
          }
        }
      ]
    }
  }
}
```